src/entities/viewer/ui/login.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import { Button } from 'shared';
 import { viewerModel } from 'entities/viewer';
 
 export const LoginButton: React.FunctionComponent = () => {
+  const navigate = useNavigate();
   return (
-    <Button onClick={() => viewerModel.loginRequested()} color="inherit">
+    <Button
+      onClick={() => {
+        viewerModel.loginRequested();
+        navigate('/profile');
```

Comment:
try to use contestants instead (shared/paths.ts ex)

Reply:
const path-profile = '/profile' ?

Reply:
did it